src/news.js

```diff
@@ -0,0 +1,13 @@
+class News {
```

Comment:
I like that you have refactored some backend logic even though it wasn't part of the instructions - it seems to me that if one ever wanted to expand this application then that would be an essential first step.